addons/mail/static/src/messaging/component/activity/activity.xml

```diff
@@ -9,14 +9,14 @@
                         <t t-if="activity.assignee">
                             <img class="o_Activity_userAvatar" t-attf-src="/web/image/res.users/{{ activity.assignee.id }}/image_128" t-att-alt="activity.assignee.nameOrDisplayName"/>
                         </t>
-                        <div class="o_Activity_icon"
+                        <div class="o_Activity_iconContainer"
                             t-att-class="{
                                 'bg-success-full': activity.state === 'planned',
                                 'bg-warning-full': activity.state === 'today',
                                 'bg-danger-full': activity.state === 'overdue',
                             }"
                         >
-                            <i class="fa" t-attf-class="{{ activity.icon }}"/>
+                            <i class="o_Activity_icon fa" t-attf-class="{{ activity.icon }}"/>
```

Comment:
not sure if it has already been mentioned before (I think it is, but it's good to remind), but that's quite fragile to have `fa` and `fa-*` part from activity field in python.

I think we should remove concern of `fa-*` from the python in the future, and instead have a declarative way to tell which kind of icon we want to have, so that JS just translates this icon name to `fa fa-*` (because using font-awesome for icons is a technical detail)

Reply:
It was some kind of done with the fontawesome upgrade commit but they decided to postpone this

Reply:
yeah, but also a remark that this is not just an issue with migration to font-awesome, but almost any changes regarding icons.

Reply:
In other words: ideally the activity icon field should contain which icon to display (eg. a phone, an email, etc. with a generic enum field), not how to display it.

This is obviously out of scope.